integration-tests/src/kusama_test_net.rs

```diff
@@ -73,7 +73,7 @@ fn default_parachains_host_configuration() -> HostConfiguration<BlockNumber> {
 		max_upward_queue_size: 1024 * 1024,
 		max_downward_message_size: 1024,
 		ump_service_total_weight: 4 * 1_000_000_000,
-		max_upward_message_size: 1024 * 1024,
+		max_upward_message_size: 50 * 1024,
```

Comment:
only wondering : why this had to change ?

Reply:
otherwise it fails on this https://github.com/paritytech/polkadot/blob/bb1bf69dfc538abd40c8d614e8bbf46580d7005a/runtime/parachains/src/configuration.rs#L393